### PR TITLE
Vim Playground Added

### PR DESCRIPTION
**Pull request by rajputpritesh1**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by domalasherpa — COMMENTED** [review]
please remove the trailing slash '/'  in the url (https://go-vim.appspot.com/) to (https://go-vim.appspot.com)

**Review by rajputpritesh1 — COMMENTED** [review]
Done

**Review by Thenlie — COMMENTED** [review]
I wonder if this would fit better under the Go heading? It seems to be more of a Go playground with Vim keybindings than a Vim playground.

**Comment by eshellman** [comment]
I agree this should go under go. go!

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -155,6 +155,7 @@
 ### Go
 
 * [Go Playground](https://play.golang.org)
+* [go-vim](https://go-vim.appspot.com)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:158]:
```suggestion
* [Go-Vim](https://go-vim.appspot.com)
```

**Pull request merged**